### DISCUSSION TITLE
New Issue When initialize an integer variable

### DISCUSSION THREAD

**Discussion by Diegomurcia** [apache/netbeans#5564] [General]
![image](https://user-images.githubusercontent.com/48791249/221368637-5fc3f774-8a4e-402f-aa14-8e07fd39476b.png)
Sometimes Netbeans allow me an integer beginning with zero, but sometimes doesn´t. Perhaps with eight and nine.
The Netbeans version it is Product Version: Apache NetBeans IDE 15 for Windows.
I hope it brings you any help.

**Comment by neilcsmith-net** [comment]
Because a number beginning with 0 is an octal number, not decimal, and therefore doesn't include the digits 8 or 9.

**Reply by Diegomurcia** [reply]
oh, thanks you. I will keep learning.
